driver/src/main/scala/utils.scala

```diff
@@ -46,7 +46,7 @@ object `package` {
 
 @deprecated(
   message = "Use [[reactivemongo.util.LazyLogger]]", since = "0.12.0")
-case class LazyLogger(logger: org.apache.logging.log4j.Logger) {
```

Comment:
Deprecated so no change

Reply:
@cchantep That doesn't make much of a difference, I'm happy to remove this if it's deprecated instead. I noticed there's already an SLF4J one. Unfortunately, this setup is causing issues now.

Reply:
By nature, a deprecated member should not be changed, but in the end removed